File: fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/error/UnknownMeetingException.java
package com.github.dedis.popstellar.utility.error;

import com.github.dedis.popstellar.R;

public class UnknownMeetingException extends UnknownEventException {

  public UnknownMeetingException(String id) {
    super("Meeting", id);
  }

  @Override
  public int getUserMessage() {
    return R.string.unknown_meeting_exception;
  }

  @Override
  public Object[] getUserMessageArguments() {
    return new Object[0];
  }
}

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/error/UnknownMeetingException.kt
package com.github.dedis.popstellar.utility.error

import com.github.dedis.popstellar.R

class UnknownMeetingException(id: String) : UnknownEventException("Meeting", id) {
    override val userMessage: Int
        get() = R.string.unknown_meeting_exception
    override val userMessageArguments: Array<Any?>
        get() = arrayOfNulls(0)
}